test_unified_integration.py

```
@@ -0,0 +1,59 @@
+        
+        for identifier, expected_type in test_cases:
+            detected_type = pipeline._detect_source_type(identifier)
+            if detected_type == expected_type:
+                print(f"✅ Source type detection: {identifier} -> {detected_type.value}")
+            else:
+                print(f"❌ Source type detection failed: {identifier} -> {detected_type.value} (expected {expected_type.value})")
+        
+        print("🎉 Unified pipeline integration test completed successfully!")
+        return True
+        
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)